Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -158,15 +176,15 @@ namespace AZ::SceneAPI::Behaviors
                 const auto thisNodeIndex = entry.first;
                 const auto meshNodeIndex = entry.second.first;
                 const auto meshNodeName = graph.GetNodeName(meshNodeIndex);
-                AZStd::string meshNodePath{ meshNodeName.GetPath() };
 
-                AZStd::string meshNodeFullName;
-                meshNodeFullName = relativeSourcePath;
-                meshNodeFullName.append("_");
-                meshNodeFullName.append(meshNodeName.GetName());
+                AZStd::string meshNodePath{ meshNodeName.GetPath() };
+                AZStd::string meshGroupName = "default_";
+                meshGroupName += scene.GetName();
```

Comment:
Why was `.append` replaced with `+=` ?

Reply:
that naming of mesh groups and asset paths with thru a bunch of changes so that is just how the code ended up after getting the code to work